### PR TITLE
Updated scenario ocp 13065 upgrade auto

### PR DESCRIPTION
**Pull request by sunilcio**
Updated scenario ocp 13065 upgrade auto

### CODE REVIEW

**Comment by sunilcio** [comment]
Execution log http://pastebin.test.redhat.com/868197

**Comment by sunilcio** [comment]
@weinliu @lyman9966 @jhou1 ptal

**Comment by weinliu** [comment]
LGTM

**Comment by jhou1** [comment]
Looks good, need to squash the commits into one before merging. @sunilcio

**Comment by lyman9966** [comment]
LGTM

File: features/upgrade/node/scc.feature [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  @admin
+  Scenario: Upgrade - Make sure seccomp part of SCC policy is kept after upgrade 
+    Given I switch to the first user
+    When I use the "node-upgrade-scc" project    
+    When I run the :get admin command with:
+      | resource      | scc     |
+      | resource name | seccomp |
+    Then the step should succeed
+    And the output should contain:
+      | seccomp |
```
Comment (liangxia) [inline features/upgrade/node/scc.feature:27]: The above 6 lines can be shortened using `Given /^(I|admin) checks? that the #{QUOTED} (\w+) exists(?: in the#{OPT_QUOTED} project)?$/ do |who, name, resource_type, namespace|`

File: features/upgrade/node/scc.feature [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/features/tierN/testdata/node/scc.yaml |
+    Then the step should succeed
+
+  # @author sunilc@redhat.com
+  # @case_id OCP-13065
+  @upgrade-check
+  @admin
+  Scenario: Upgrade - Make sure seccomp part of SCC policy is kept after upgrade 
+    Given I switch to the first user
```
Comment (pruan-rht) [inline features/upgrade/node/scc.feature:20]: @sunilcio I think you mean to do `Given I switch to cluster admin pseudo user`

File: features/upgrade/node/scc.feature (hunk 1 above)
Comment (pruan-rht) [reply features/upgrade/node/scc.feature:27]: or `And the expression should be true> scc('seccomp2').exists?`

File: features/upgrade/node/scc.feature (hunk 2 above)
Comment (sunilcio) [reply features/upgrade/node/scc.feature]: @pruan-rht yeah, correct. updated :)

**Pull request closed without merging**